src/lib/raidbots/getTalentTrees.ts

```diff
@@ -9,13 +9,20 @@ export async function getTalentTrees(scope: Scope = "live") {
         ({ result: response, time } = await measuredPromise(
             fetch(
                 `https://www.raidbots.com/static/data/${scope}/talents.json`,
-                // The data is larger than 2MB, so we're not allowed to cache it.
-                // { cache: "no-store" },
+                {
+                    // Note: Data is ~4MB, exceeds Next.js 2MB data cache limit,
+                    // but HTTP fetch cache still applies for performance
+                    next: {
+                        revalidate: 86400, // 24 hours - talent data changes infrequently
+                        tags: [`raidbots-talents-${scope}`],
+                    },
+                },
```

Comment:
If we try to cache it, will nextjs not just complain about the size of it? Have you tried it? have you seen any warnings?